examples/seq2seq/seq2seq_trainer.py

```diff
@@ -64,6 +58,17 @@ def __init__(self, config=None, data_args=None, *args, **kwargs):
                 f"The `config.pad_token_id` is `None`. Using `config.eos_token_id` = {self.config.eos_token_id} for padding.."
             )
 
+        if self.args.label_smoothing == 0:
+            self.loss_fn = torch.nn.CrossEntropyLoss(ignore_index=self.config.pad_token_id)
+        else:
+            # dynamically import label_smoothed_nll_loss
+            try:
```

Comment:
We moved all seq2seq scripts/tests to not need the try/except workaround, and just `import from utils`, see:
https://github.com/huggingface/transformers/pull/7274/files#diff-36ebee2221416f906bccc099a8958f6814f51a1dc8f2b07a75ffd19bd0d1ddf6
this script looks like a new file that was copied from some old style code.

Of course, your PR didn't introduce it, but just as well let's use the clean new style.

Reply:
Thanks for letting me know! -> will open a PR to correct it :-)

Reply:
Pinged you on the PR :-) https://github.com/huggingface/transformers/pull/8254